docs/using_third_party_states.ipynb

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.        def apply_gate_to_subspace(gate:np.ndarray, state:np.ndarray, indices:np.ndarray):
</pre><p>Can you put this and <code>apply_gate</code>  outside of <code>apply_cirq_gate</code> ?   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asciineuron/bgls/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>